docker/start.sh

```diff
@@ -0,0 +1,5 @@
+set -ex
```

Comment:
We want a single entry point for any service that might be needed for integration tests.

A Makefile will be OK with some parameteriszation for the service

Could be parameterized as follows:

`make build_graphdb`
`make start_graphdb`

Reply:
Yes, that would be a good thing, I have made changes accordingly.